requirements.txt

```diff
@@ -1,9 +1,9 @@
-aiofiles>=0.8.0
-async_lru~=1.0
-galaxy_importer>=0.4.5
-GitPython>=3.1.24
-jsonschema~=4.4
-packaging
-pulpcore>=3.18
-PyYAML
-semantic_version
+aiofiles>=0.8.0,<0.9
+async_lru>=1.0,<1.1
+galaxy_importer>=0.4.5,<0.5
+GitPython>=3.1.24,<3.2
+jsonschema>=4.4,<4.7
```

Comment:
I think in practice we do support much older ones. We kind of consolidated on 4.4 project-wide, but the lower bound could be lower. This was already set though, so I'm not asking for a change here, just a FYI.

Reply:
https://github.com/pulp/pulp_container/issues/772